test/optimize/test_optimize.py

```diff
@@ -675,3 +678,22 @@ def test_assign_ids():
 
     ids = assign_ids(n_starts=n_starts, ids=None, result=result)
     assert ids == [str(i) for i in range(n_starts, n_starts * 2)]
+
+
+def test_laplace_approximation_log_evidence():
+    prob = pypesto.Problem(
+        objective=rosen_for_sensi(max_sensi_order=2)["obj"],
+        lb=0 * np.ones((1, 2)),
+        ub=1 * np.ones((1, 2)),
+    )
+
+    # hess
+    result = optimize.minimize(
+        problem=prob,
+        n_starts=1,
+        progress_bar=False,
+    )
+    log_evidence = laplace_approximation_log_evidence(
+        prob, result.optimize_result.x[0]
+    )
+    assert isinstance(log_evidence, float)
```

Comment:
is the a possibility to assert more than just a float return? Would probably be fine to adjust certain values manually to create a predicatble result?